libs/commons/common2.ml

```diff
@@ -1396,6 +1400,21 @@ let int_of_all s =
   then int_of_octal s
   else int_of_string s
 
+let int64_of_string_opt s =
```

Comment:
move in Int64_.ml

Reply:
`int64_of_string_opt` relies on `Common.matched1`, and `Common.float_of_string_opt` relies on `int64_of_string_opt`, so this is kind of painful to do. I elected to keep it here just to keep the dependencies simple.